sdk/src/wallet/account/operations/balance.rs

```diff
@@ -19,10 +20,13 @@ use crate::{
     },
 };
 
-impl Account {
-    /// Get the balance of the account.
-    pub async fn balance(&self) -> crate::wallet::Result<Balance> {
-        log::debug!("[BALANCE] balance");
+impl<S: 'static + SecretManage> Account<S>
+where
+    Error: From<S::Error>,
+{
+    /// Get the AccountBalance
```

Comment:
```suggestion
    /// Get the balance of the account.
```
AccountBalance is no more